docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - POSTGRES_PASSWORD=$POSTGRES_PASSWORD
       - POSTGRES_DB=$POSTGRES_DATABASE
     ports:
-      - "$POSTGRES_LOCAL_PORT:$POSTGRES_DOCKER_PORT"
+      - "5435:$POSTGRES_DOCKER_PORT"
```

Comment:
why variable is replaced with hardcoded value?

Reply:
it was done due to error, which I got when I want to run my application. Error was due to PID, so, when I killed this PID, I forgot to change back to holders.